monitoring/prometheus/rules/alerts.yml

```
@@ -0,0 +1,87 @@
+        annotations:
+          summary: "Redis is down"
+          description: "Redis instance has been down for more than 2 minutes"
+
+      - alert: RedisHighMemoryUsage
+        expr: redis_memory_used_bytes / redis_config_maxmemory * 100 > 90
+        for: 5m
+        labels:
+          severity: warning
+        annotations:
+          summary: "Redis memory usage high"
+          description: "Redis memory usage is above 90%"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)